src/Devices/PHProbe.cpp

```diff
@@ -42,13 +43,13 @@ void PHProbe::clearCalibration() {
 void PHProbe::sendSlopeRequest() {
   // Sending request for Calibration Slope
   Serial1.print(F("SLOPE,?\r"));
-  strncpy_P(slopeResponse, (PGM_P)F("       Slope requested!"), sizeof(slopeResponse));  // Flawfinder: ignore
+  strscpy_P(slopeResponse, (PGM_P)F("       Slope requested!"), sizeof(slopeResponse));  // Flawfinder: ignore
```

Comment:
Do we still need `Flawfinder: ignore` on this line (and following ones)? I wonder if it is finding potential bugs with public library functions, but since we have our own functions it doesn't try to find problems. On the other hand, the `_P` version would probably not be known.

Reply:
You're right, this flag is no longer needed.